backend/fms_core/tests/test_template_importers/sample_submission_test.py

```diff
@@ -83,7 +83,7 @@ def test_import(self):
             derived_sample_id = DerivedBySample.objects.filter(sample_id=sample_obj.id).first().derived_sample_id
             biosample = DerivedSample.objects.get(id=derived_sample_id).biosample
             if biosample.individual.is_generic:
-                self.assertEqual(biosample.individual.name, generic_individual)
+                self.assertIn(generic_individual_prefix, biosample.individual.name)
```

Comment:
we should instead be asserting that the individual name starts with prefix, not that it just contains the prefix.